### PR TITLE
Add dbcs to racecar repo

### PR DESCRIPTION
**Pull request by samparent97**
Dbcs moved from the `can_dbc` repo to racecar repo.

### CODE REVIEW

File: dbcs/PT_CAN.dbc
Comment (BlakeFreer) [inline dbcs/PT_CAN.dbc:1]: I consider the dbcs to be part of
Firmware (they are fundamentally just an inter-ecu message protocol) and think they should go in the firmware directory (firmware/dbc).

Also consider that we want to automate the Dbc-to-code step, so cmake will call the code generator, and cmake is designed to work on subdirectories

File: dbcs/legacy/CAN_AMKInverter.dbc
[the remark this answers is not in the document]
Comment (samparent97) [reply dbcs/legacy/CAN_AMKInverter.dbc:1]: Thought about leaving them out. Thought it made most sense to keep this changeset as atomic as possible: move the dbcs from the can_dbc repo to the racecar repo.

[the remark this answers is not in the document]
Comment (BlakeFreer) [reply dbcs/legacy/CAN_AMKInverter.dbc:1]: Sounds good, I'd like to have a pr in the future to remove unused files though. Maybe once we get working firmware running

File: dbcs/PT_CAN.dbc
Comment (BlakeFreer) [reply dbcs/PT_CAN.dbc:1]: Will figure out later

**Review by BlakeFreer — APPROVED** [review]

**Pull request merged**